controllers/ControllerAdmin.php

```diff
@@ -73,19 +77,20 @@ public function checkMailProfile(){
 
 	public function checkPassProfile(){
 		$params = $this->params['post'];
+		$key = (new Session)->getKey('user');
 
 		$this->checkForm($params);
 
 		if($params['password'] != $params['confirme']){
 			$valid = true;
-			echo $this->twig->render('profile.twig', ['valid' => $valid, 'logged'=> self::LOGGED, 'user'=> $_SESSION['user']['pseudo'], 'value'=> $_SESSION['user']['email']]);
+			echo $this->twig->render('profile.twig', ['valid' => $valid, 'logged'=> self::LOGGED, 'user'=> $key['pseudo'], 'value'=> $key['email']]);
```

Comment:
Codacy found an issue: [All output should be run through an escaping function (see the Security sections in the WordPress Developer Handbooks), found '$this'.](https://app.codacy.com/manual/PatOpen/OC_Blog/pullRequest?prid=5982252)